translator.go

```diff
@@ -608,11 +608,19 @@ func (t *DefaultAtomTranslator) translateItemUpdatedParsed(entry *atom.Entry) (u
 }
 
 func (t *DefaultAtomTranslator) translateItemPublished(entry *atom.Entry) (updated string) {
-	return entry.Published
+	updated = entry.Published
```

Comment:
While not your fault, it would be nice to rename the variable `updated` to `published` to know what is going on here. I was slightly puzzled reading the code :)

Reply:
Done :)